### ISSUE TITLE
Link to contamehistorias.pt

### ISSUE THREAD

**Issue by igobranco** [arquivo/pwa-technologies#976]
**What is the URL that originated the issue?**
http://arquivo.pt/
http://arquivo.pt/page/search

**What happened?**
Add a button to contamehistorias.pt on the Arquivo.pt search pages and images.

In the middle of Images and Advanced search buttons.

When clicking on the button, add a modal box that informs the user that he is going to an external service, with cancel/ok like we have on the options on wayback.

![image](https://user-images.githubusercontent.com/67018/86479593-5d5ca580-bd44-11ea-8aac-4ef70ad8e1c7.png)

**Comment by dcgomes** [comment]
Present disclaimer light-house before jumping to CMH UI (similar to behavior on "Replay with old browser" feature).
Text could be "You are leaving Arquivo.pt to ContaMeHistorias.pt service"

**Comment by igobranco** [comment]
To implement the option 1.

**Comment by dcgomes** [comment]
contamehistorias.pt/arquivopt/search?query=$QUERY

**Comment by dcgomes** [comment]
Message after click should be:

"Vai sair do Arquivo.pt para o <a href="https://ContaMeHistorias.pt>ContaMeHistorias.pt</a>"

Also apply same format to similar messages:

Completar página: "Vai sair do Arquivo.pt para o <a href="http://oldweb.today>OldWeb.today</a>"

**Comment by hmendes** [comment]
Solved.

**Comment by dcgomes** [comment]
- Change text on button "Narrativa Temporal" to "Narrativa temporal"
- Change message as in my previous comment: 
"Vai sair do Arquivo.pt para o <a `href="https://ContaMeHistorias.pt>ContaMeHistorias.pt</a>"`

**Comment by hmendes** [comment]
This needs to be edited in the markup/html. @amourao Can you help, please?

**Comment by amourao** [comment]
@hmendes Link with id "ContaMeHistoriasModalOutlink" needs to be re-styled as it shows up as white on a white background.
![image](https://user-images.githubusercontent.com/2866295/96877227-4b2c8180-1471-11eb-89e5-54325d9a5682.png)

**Comment by amourao** [comment]
> Message after click should be:
> 
> `Completar página: "Vai sair do Arquivo.pt para o <a href="http://oldweb.today>OldWeb.today</a>" `

@dcgomes Opened separate issue, as that is a part of pywb, not webapp #1037

**Comment by PedroG1515** [comment]
Problems in mobile

![1](https://user-images.githubusercontent.com/25795364/97550701-e9689c00-19c9-11eb-8bb9-c19b53b7fc71.JPG)

**Comment by dcgomes** [comment]
@amourao add flag to hide/show "Narrativa temporal"

**Comment by dcgomes** [comment]
- ContaMeHistorias.pt must display a "Searching information for you... it may take a few moments" icon when service is requested directly through URL so that users: don't think that the system is down after clicking on "Narrativa temporal" on Arquivo.pt
http://contamehistorias.pt/arquivopt/search?query=rui%20unas

- Queries not in cache of CMH may overload Arquivo.pt

- Why is CMH making 12 searches per query in parallel with pairs of searches by site:
2020-10-27T18:59:54,237 INFO  NutchBean - [NuchBean][search] Query[date:"20090102000000-20181231000000" torres novas site:"www.ionline.pt" site:"www.jn.pt" type:"html"] Queryservers response time: 3.144492685

2020-10-27T18:59:54,875 INFO  NutchBean - [NuchBean][search] Query[date:"20090102000000-20181231000000" torres novas site:"dnoticias.pt" site:"sicnoticias.sapo.pt" type:"html"] Queryservers response time: 3.771710164

2020-10-27T18:59:55,119 INFO  NutchBean - [NuchBean][search] Query[date:"20090102000000-20181231000000" torres novas site:"noticias.sapo.pt" site:"www.cmjornal.pt" type:"html"] Queryservers response time: 4.023393386

2020-10-27T18:59:55,929 INFO  NutchBean - [NuchBean][search] Query[date:"20090102000000-20181231000000" torres novas site:"www.rtp.pt" site:"news.google.pt" type:"html"] Queryservers response time: 4.797592329

- We must accept that CMH responses will be slow.
- @vitgou Apache QoS thresholds of Arquivo.pt must be reviewed to return "HTTP 429 Too many requests" at lower rates. Check at 10 seconds intervals?
- Could make less parallel requests to Arquivo.pt at expense of delaying responses?
- Total time of sequential requests could be lower than parallel because Arquivo.pt caches responses from first requests.
- Why 2 sites per request? Have you tried 1 site per request?

**Comment by pcabritafccn** [comment]
This issue is now solved.

**Comment by dcgomes** [comment]
CMH team will evaluate proposals to reduce load.

**Comment by vitgou** [comment]
Ready, waiting for CMH team.

**Comment by vitgou** [comment]
The link is ready and CMH server is available. Check arquivo/pwa-technologies#1155 for deployment
